motr/client.h

```diff
@@ -50,7 +50,7 @@
  *
  *     - Motr-based block device (part of BOMO);
  *
- *     - exascale E10 stack (see eiow.org).
+ *     - exascale E10 stack (see https://opensfs.org).
```

Comment:
opensfs is not related to E10 work, domain of eiow is no longer there, let's remove it instead of changing it opensfs.

Reply:
@sainarasim and @nikitadanilov , is there any relevant E10 stack webpages that might still be useful to link? If not, then @madhavemuri is right and we should just remove this line altogether.

Reply:
Yes, let's remove it.